notebooks/book2/02/sub_super_gauss_plot.ipynb

```diff
@@ -0,0 +1,461 @@
+{
```

Comment:
<p><code>xlabel</code> (use $x$) &amp; <code>ylabel</code> (use $p(x)$) missing  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Okay, I will add labels to the first figure